### DISCUSSION TITLE
Commercial Use

### DISCUSSION THREAD

**Discussion by braxtonj** [logpai/loghub#52] [Q&A] [answered]
Hey Loghub team! 

Wondering who to reach out to for Commercial use.  Our team is interested in using a number datasets for demo purposes but not seeing contact information or licensing outside Academic + Research.

**Comment by zhujiem** [comment, marked as answer]
Could you please send a request email to info@logpai.com? Please include the company name, and what datasets you plan to use for commercial.

**Reply by braxtonj** [reply]
Perfect, just sent out an email.  Appreciate the quick response!
